src/todomanager/popup.tsx

```diff
@@ -42,36 +46,101 @@ function Popup() {
     );
   };
 
+  const deleteTodo = (id: string) => {
+    setTodos(todos.filter(todo => todo.id !== id));
+  };
+
+  const handleEditTodo = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (editTodo) {
+      setEditTodo({ ...editTodo, content: e.target.value });
+    }
+  };
+  const startEditTodo = (todo: Todo) => {
+    setEditTodo({ id: todo.id, content: todo.content });
+  };
+
+  const saveEditTodo = () => {
```

Comment:
```suggestion
  const handleSaveEditTodo = () => {
```

Request: 다른 부분도 통일 시켜주세요

Reply:
통일했습니다! [f22fcdc](https://github.com/ToyMakers/todo-mark/pull/11/commits/f22fcdc8fca0c48e273709984211cf38f6f9b334)